lib/config/deviceconfiguration.pb.go

```diff
@@ -0,0 +1,212 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: lib/config/deviceconfiguration.proto
+
+package config
+
+import (
+	fmt "fmt"
+	proto "github.com/gogo/protobuf/proto"
+	github_com_syncthing_syncthing_lib_protocol "github.com/syncthing/syncthing/lib/protocol"
+	protocol "github.com/syncthing/syncthing/lib/protocol"
```

Comment:
Something is weird here

Reply:
Auto-generated code is auto-generated.
One is added for custom type conversion, the other one is added as a dependency of the proto type.

Reply:
Right, that's the `ext.device_id` thing. What's the difference to just using `(gogoproto.customtype) = "DeviceID"` like in bep.proto?

Reply:
It would have to be a fully qualified path, so
`(gogoproto.customtype = "github.com/syncthing/syncthing/lib/protocol.DeviceID"`